.babelrc

```diff
@@ -1,18 +1,41 @@
 {
-  "env":{
+  "env": {
     "development": {
-      "presets": ["next/babel"]
+      "presets": [
+        "next/babel"
+      ]
     },
     "production": {
-      "presets": ["next/babel", "minify"]
+      "presets": [
+        "next/babel",
+        [
+          "minify",
+          {
+            "builtIns": false
```

Comment:
Suggestion from: https://github.com/babel/minify/issues/904#issuecomment-464953535 (should be temporal)